CONTRIBUTING.md

```diff
@@ -7,3 +7,6 @@ To ensure a consistent markdown style, we use [dprint](https://dprint.dev) to ch
 ```shell
 dprint fmt
 ```
+
+Use the [official documentation](https://dprint.dev/setup/) to setup dprint.
+The configuration can be found [here](https://github.com/bakdata/ci-templates/blob/main/dprint.json).
```

Comment:
Should we maybe say something about what @yannick-roeder commented ? using prettier in vs code or something like that ?

Reply:
Sure does not hurt to mention it for commonly used editors

Reply:
@yannick-roeder Do you have the configuration for that? I have never used Prettier before.